### DISCUSSION TITLE
Latex commands replaced by Dummy results in spelling error

### DISCUSSION THREAD

**Discussion by dsegerss** [ltex-plus/vscode-ltex-plus#87] [Q&A]
Hi,
I just started using LTEX. When using the Swedish "sv" language setting, the Dummy placeholders that are added instead of \ref, \cite etc. are marked with HUNSPELL-RULE. I can of-course add Dummy0, Dummy1 etc. to the user dictionary, but I assume this is not how it's supposed to be. I would appreciate advice regarding how this should be handled to ignore all Dummy placeholders in a document.

**Comment by spitzerd** [comment]
Right now, LTeX+ has special Dummy strings for German and French only. All other languages, including English, use Dummy0, Dummy1 , et cetera.

**Comment by spitzerd** [comment]
I created issue #89 for you and I will create a special rule for Swedish.
